pages/_app.js

```diff
@@ -1,13 +1,15 @@
 import GlobalStyles from "../components/GlobalStyles";
 import { useLocalStorage } from "../helpers/hooks.js";
 import moviesDb from "../helpers/db.json";
+import Header from "../components/Header";
 
 function MyApp({ Component, pageProps }) {
   const [movies, setMovies] = useLocalStorage("movies", moviesDb);
 
   return (
     <>
       <GlobalStyles />
+      <Header />
```

Comment:
If you use your header component here, it will be visible on every page.
That's probably why you wrote a condition with if/else. My recommendation: use it where you need it. 🖖

```suggestion

```